### PR TITLE
[1.18] Change default target vocab name in model folder to vocab.trg.0.json

### PR DESCRIPTION
**Pull request by fhieber**
#### Pull Request Checklist ##
- [x] Changes are complete (if posting work-in-progress code, prefix your pull request title with '[WIP]'
until you can check this box.
- [x] Unit tests pass (`pytest`)
- [x] Were system tests modified? If so did you run these at least 5 times to account for the variation across runs?
- [x] System tests pass (`pytest test/system`)
- [x] Passed code style checking (`./style-check.sh`)
- [x] You have considered writing a test
- [x] Updated major/minor version in `sockeye/__init__.py`. Major version bump if this is a backwards incompatible change.
- [x] Updated CHANGELOG.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by davvil** [comment]
I have no issues with the implementation itself, but why do we want it? Just for consistency with the source vocabs?

**Comment by fhieber** [comment]
Mostly for parallelism of vocabulary names. Also, if we would add target factors we may be able to avoid a MV bump.

File: sockeye/constants.py [hunk 1]
```diff
@@ -195,7 +195,8 @@
 JSON_SUFFIX = ".json"
 VOCAB_SRC_PREFIX = "vocab.src"
 VOCAB_SRC_NAME = VOCAB_SRC_PREFIX + ".%d" + JSON_SUFFIX
-VOCAB_TRG_NAME = "vocab.trg" + JSON_SUFFIX
```
Comment (mjpost) [inline sockeye/constants.py]: As long as we're changing this, shouldn't it be ".tgt"?

Comment (fhieber) [reply sockeye/constants.py]: I am a big 'trg' believer :)

**Pull request merged**